rholang/src/test/scala/coop/rchain/rholang/interpreter/PrettyPrinterTest.scala

```diff
@@ -113,6 +113,29 @@ class ProcPrinterSpec extends FlatSpec with Matchers {
     result shouldBe target
   }
 
+  it should "not drop asterisk in front of variable that is sent" in {
+    val source = Par(
+      news = Seq(
+        New(
+          1,
+          Par(
+            sends = Seq(
+              Send(
+                data = Seq(Par(exprs = Seq(EVar(BoundVar(0)))))
+              )
+            )
+          )
+        )
+      )
+    )
+    val result = PrettyPrinter().buildString(source)
+    val target =
+      """new x0 in {
+        |  @{Nil}!(*x0)
+        |}""".stripMargin
+    result shouldBe target
+  }
```

Comment:
Can you add a test for:
```scala
for(@x <- @NIl) { @Nil!(x) } 
```
?
Not every bound variable should be preceded with an asterix - only names should be. Although I don't it's possible to know just from the protobuff itself - there's no way to tell in which context the variable was bound :/